crates/bitwarden-vault/src/cipher/ssh_key.rs

```diff
@@ -0,0 +1,43 @@
+use bitwarden_crypto::{
+    CryptoError, EncString, KeyDecryptable, KeyEncryptable, SymmetricCryptoKey,
+};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SshKey {
+    pub private_key: Option<EncString>,
+    pub public_key: Option<EncString>,
+    pub fingerprint: Option<EncString>,
```

Comment:
**question:** Do we need to persist `public_key` and `fingerprint`? We could derive them from the private key.

Reply:
We need these on clients that don't have SDK support (yet). I do think that we can drop these later on when we have SDK integrated everywhere.

Reply:
^ We can remove them with cipher versioning once we don't need them anymore. Resolving this.